apps/sel4test-tests/src/tests/fpu.c

```diff
@@ -187,4 +187,4 @@ int smp_test_fpu(env_t env)
     return sel4test_get_result();
 }
 DEFINE_TEST(FPU0002, "Test FPU remain valid across core migration", smp_test_fpu,
-            config_set(CONFIG_MAX_NUM_NODES) &&config_set(CONFIG_HAVE_TIMER) &&CONFIG_MAX_NUM_NODES > 1)
+            config_set(CONFIG_HAVE_TIMER) &&CONFIG_MAX_NUM_NODES > 1)
```

Comment:
maybe improve the readability also 
```suggestion
            config_set(CONFIG_HAVE_TIMER) && (CONFIG_MAX_NUM_NODES > 1))
```

Reply:
Let's try that and see if the style checker accepts it

Reply:
Unforunately (and annoyingly) it insists on removing the space after `&&`, but I think the parentheses do help in this case, so I'll add them.